### PR TITLE
Added section about virt system roles (jsc#SLE-17282)

### PR DESCRIPTION
**Pull request by tbazant**
### Description

Virtualization system roles were modified. This update mentions them and adds brief description.
Rendered version is attached 
[cha-vt-installation_color_en.pdf](https://github.com/SUSE/doc-sle/files/5932228/cha-vt-installation_color_en.pdf)

### Are there any relevant issues/feature requests?

* jsc#SLE-17282

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ---------- 
| [x] 15 SP3  | (`master` only)   
| [ ] 15 SP2  | [ ] 
| [ ] 15 SP1  | [ ] 
| [ ] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

**Review by aginies — COMMENTED** [review]
sounds ok .

File: xml/vt_installation.xml [hunk 1]
```diff
@@ -14,106 +14,160 @@
   </dm:docmanager>
  </info>
  <para>
-  Depending on the scope of the installation, none of the virtualization tools
-  may be installed on your system. They will be automatically installed when
-  configuring the hypervisor with the &yast; module <menuchoice>
-  <guimenu>Virtualization</guimenu> <guimenu>Install Hypervisor and
-  Tools</guimenu></menuchoice>. In case this module is not available in &yast;,
-  install the package <package>yast2-vm</package>.
+  You can install virtualization tools required to run a &vmhost; in two ways:
```
Comment (lproven) [inline xml/vt_installation.xml:17]:
```suggestion
  You can install the virtualization tools required to run a &vmhost; in two ways:
```

File: xml/vt_installation.xml [hunk 2]
```diff
@@ -14,106 +14,160 @@
...
+ </itemizedlist>
 
- <sect1 xml:id="sec-vt-installation-kvm">
-  <title>Installing &kvm;</title>
-
+ <sect1 xml:id="vt-installation-system-role">
+  <title>Specifying a system role</title>
   <para>
-   To install &kvm; and KVM tools, proceed as follows:
+   You can install all required virtualization tools already during the host
```
Comment (lproven) [inline xml/vt_installation.xml:35]:
```suggestion
   You can install all the tools required for virtualization during the host
```

File: xml/vt_installation.xml [hunk 3]
```diff
@@ -14,106 +14,160 @@
...
+   &productname; installation. During installation steps
```
Comment (lproven) [inline xml/vt_installation.xml:36]:
```suggestion
   &productname; installation. During the installation steps
```

File: xml/vt_installation.xml [hunk 4]
```diff
@@ -14,106 +14,160 @@
...
+    <imageobject role="html">
+     <imagedata fileref="virt-system-roles.png" width="75%"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   Here you can select one of <guimenu>KVM Virtualization Host</guimenu> or
+   <guimenu>Xen Virtualization Host</guimenu> roles. The appropriate software
+   selection and setup will be automatically aligned during the host
+   installation.
```
Comment (lproven) [inline xml/vt_installation.xml:55]:
```suggestion
   Here you can select either the <guimenu>KVM Virtualization Host</guimenu> or
   <guimenu>Xen Virtualization Host</guimenu> roles. The appropriate software
   selection and setup will be automatically performed during the OS
   installation.
```

File: xml/vt_installation.xml [hunk 5]
```diff
@@ -14,106 +14,160 @@
...
-     After the setup has been finished, you need to reboot the machine with
-     the <emphasis>&xen; kernel</emphasis>.
-    </para>
-    <tip>
-     <title>Default boot kernel</title>
+  <sect2 xml:id="sec-vt-installation-kvm">
+   <title>Installing &kvm;</title>
+
+   <para>
+    To install &kvm; and KVM tools, proceed as follows:
```
Comment (lproven) [inline xml/vt_installation.xml:81]:
```suggestion
    To install &kvm; and the KVM tools, proceed as follows:
```

File: xml/vt_installation.xml [hunk 6]
```diff
@@ -14,106 +14,160 @@
...
+   </para>
+
+   <procedure>
+    <step>
      <para>
-      If everything works as expected, change the default boot kernel with &yast; and make the &xen;-enabled kernel the default.
-      For more information about changing the default kernel, see <xref linkend="sec-grub2-yast2-config"/>.
+      Verify that the <package>yast2-vm</package> package is installed. This package
+      is &yast;'s configuration tool that simplifies the installation of virtualization
+      hypervisors.
```
Comment (lproven) [inline xml/vt_installation.xml:89]:
```suggestion
      is &yast;'s configuration tool that simplifies the installation of hypervisors.
```

Comment (lproven) [reply xml/vt_installation.xml:89]: There's no other kind of hypervisor than a virtualization hypervisor, so I think we can drop the qualifying adjective here.

**Review by lproven — CHANGES_REQUESTED** [review]

**Pull request merged**